dockers/docker-database/database_config.json.j2

```diff
@@ -15,6 +16,14 @@
             "unix_socket_path": "/var/run/redis-chassis/redis_chassis.sock",
             "persistence_for_warm_boot" : "yes"
         }
+{% if include_remote_db %}
+        ,"remote_redis":{
```

Comment:
>, [](http://example.com/codeflow?start=8&length=1)

The `,` convention will be weird. Would you be able to move it to previous line, I am ok with this j2 file mess up.